Tests/AGSnapshotHelperTests/FBSnapshotTestCase+AGSnapshotHelperTests.m

```diff
@@ -0,0 +1,56 @@
+//
+//  FBSnapshotTestCase+AGSnapshotHelperTests.m
+//  AGSnapshotHelper
+//
+//  Created by Adam Grzegorowski on 15/01/16.
+//  Copyright © 2016 allegro. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import <FBSnapshotTestCase/FBSnapshotTestController.h>
+#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+#import <AGSnapshotHelper/FBSnapshotTestCase+AGSnapshotHelper.h>
+
+static CGRect AGViewFrame = (CGRect){{0.0f, 0.0f}, {50.0f, 50.0f}};
+
+@interface FBSnapshotTestCase_AGSnapshotHelperTests : FBSnapshotTestCase
+@end
+
+@implementation FBSnapshotTestCase_AGSnapshotHelperTests
+
+- (void)testTestViewOfContentSizeCategoryWithDisabledRecodedModeShouldCompareView {
+
+    // Arrange
+    self.recordMode = NO;
+
+    // Act & Assert
+    XCTAssertNoThrow([self testView:[self exampleView] contentSizeCategory:UIContentSizeCategoryLarge]);
+}
+
+- (void)testTestLayerOfContentSizeCategoryWithDisabledRecodedModeShouldCompareView {
+
+    // Arrange
+    self.recordMode = NO;
+
+    // Act & Assert
+    XCTAssertNoThrow([self testLayer:[self exampleLayer] contentSizeCategory:UIContentSizeCategoryLarge]);
+}
+
+#pragma mark - Helper methods
+
+- (UIView *)exampleView {
+    UIView *exampleView = [[UIView alloc] initWithFrame:AGViewFrame];
+    exampleView.backgroundColor = [UIColor redColor];
+
+    return exampleView;
```

Comment:
Same UILabel doesn't have any Dynamic Type logic, so this would require additional class.
Those tests will be refactored anyway, so we could provide such changes by the way.